doc/tests/empty.rst

```diff
@@ -1,11 +1,19 @@
 ``empty``
 =========
 
-``empty`` checks if a variable is empty:
+``empty`` checks if a variable is
+ - an empty string
+ - an empty array
+ - an empty hash
+ - exactly ``false``
+ - exactly ``null``
```

Comment:
Before the bulleted list clarified that "an instance of Countable_ and has a count_ of zero" go together, but removing that makes CSV format work.